Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -55,16 +55,17 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
     }
 
     @objc open var isFullscreen: Bool = false
-    private var isChromeless: Bool { options.bool(kChromeless) }
+    public var chromelessMode: Bool = false
     
     public required init(options: Options = [:], layerComposer: LayerComposer) {
         Logger.logDebug("loading with \(options)", scope: "\(type(of: self))")
 
         self.options = options
         self.layerComposer = layerComposer
         super.init()
-        
-        if !isChromeless {
+        self.updateChromelessMode(with: options)
+
+        if !chromelessMode {
```

Comment:
I guess what we actually want to do here is to enable/disable de gesture recognizers based on the Chromeless state. The gestures should always be added on initialization, we just want to disable it at a particular moment.

Reply:
Actually on the init the `options` didSet is not called. 
Reference:  Swift Documentation 
[_willSet and didSet observers are not called when a property is first initialized. They are only called when the property’s value is set outside of an initialization context._](https://docs.swift.org/swift-book/LanguageGuide/Properties.html#//apple_ref/doc/uid/TP40014097-CH14-XID_368)